prov/sockets/include/sock_util.h

```diff
@@ -74,6 +74,8 @@ static inline void *sock_mremap(void *old_address, size_t old_size,
 	return (void *) -1;
 #elif defined __FreeBSD__
 	return (void *) -1;
+#elif defined _WIN32
+	return (void *) -1;
 #else
 	return mremap(old_address, old_size, new_size, 0);
 #endif
```

Comment:
@jithinjosepkl - can you please look at defining mremep as part of the osd.h headers, so we can avoid the ifdef's here?